smart_control/simulator/simulator_flexible_floor_plan_test.py

```diff
@@ -219,24 +222,38 @@ def _create_small_building(
         conductivity=2.0, heat_capacity=500.0, density=1800.0
     )
     building_exterior_properties = building_py.MaterialProperties(
-        conductivity=0.05, heat_capacity=500.0, density=3000.0
+        conductivity=1.0, heat_capacity=500.0, density=3000.0
```

Comment:
Is this value change to make the exterior properties more realistic (issue with the value being used previously)?

This question underscores the helpfulness of including some `MaterialProperties` lookup values / list of values for common materials in the docs.